src/React/Basic.purs

```diff
@@ -8,13 +8,15 @@ module React.Basic
   , fragmentKeyed
   , JSX
   , ReactComponent
+  , ReactComponentInstance
+  , react
```

Comment:
I'm thinking about the import story here. The ideal for me would be if qualified imports were assumed, so the code could be written as:

```purescript
module MyComponent (component, State, Props) where

import React.Basic as React

type State = {}
type Props = {}

component :: React.Component Props
component = React.component {..}
```

I find the current naming a bit inconsistent (some names are prefixed with `React` and some aren't). What do you think?

Reply:
That seems fine to me, we use that a bit as well. Pretty sure this already needs to be a major version bump anyway.

Reply:
I guess it could also be added with a backward compatible synonym (`type ReactComponent = Component`), same as `react` -> `component`.